### PR TITLE
Fix https:// URLs and update the readme

### PR DESCRIPTION
**Pull request by ronkorving**
Please test on https://ronkorving.github.io/wizcorpsite/

Changes:

- Grunt is now part of the package, and runs pre-commit.
- http URLs have been changed to https as far as possible and required.
- ReadMe instructions update.
- Various code style improvements.

cc @everydayfoxlife @baptistebrunet

### CODE REVIEW

**Comment by everydayfoxlife** [comment]
I think the Publications feed for the Japanese version of the site needs to have the Tumblr URL fixed as well. I'm getting this error:
image

**Comment by ronkorving** [comment]
We're working on that right now.

**Comment by ronkorving** [comment]
That should now be fixed.

File: index.html [hunk 1]
```diff
@@ -504,7 +504,7 @@ <h3 class="section-content-title">
                                                 <div class="controls controls-row">
                                                     <span class="wpcf7-form-control-wrap your-Budget"><input type="text" name="your-Budget" value="" size="40" class="wpcf7-form-control wpcf7-text wpcf7-validates-as-required span6" aria-required="true" aria-invalid="false" placeholder="Budget*" /></span><span class="wpcf7-form-control-wrap your-Timeframe"><input type="text" name="your-Timeframe" value="" size="40" class="wpcf7-form-control wpcf7-text wpcf7-validates-as-required span6" aria-required="true" aria-invalid="false" placeholder="Timeframe*" /></span>
                                                 </div>
-                                                <p><span class="wpcf7-form-control-wrap your-message"><textarea name="your-message" cols="40" rows="5" class="wpcf7-form-control wpcf7-textarea wpcf7-validates-as-required" aria-required="true" aria-invalid="false" placeholder="Message*"></textarea></span><br />
```
Comment (everydayfoxlife) [inline index.html]: is there a reason to keep this commented out code? it looks like a wordpress contact form plugin, but the site is not using wordpress...

Comment (ronkorving) [inline index.html]: I'll get rid of it.

**Comment by everydayfoxlife** [comment]
LGTM!

**Comment by ronkorving** [comment]
Thanks!

**Pull request merged**